baseplate/context/sqlalchemy.py

```diff
@@ -95,15 +95,15 @@ class SQLAlchemySessionContextFactory(SQLAlchemyEngineContextFactory):
     :param sqlalchemy.engine.Engine engine: A configured SQLAlchemy engine.
 
     """
-    def make_object_for_context(self, name, server_span):
+    def make_object_for_context(self, name, span):
         engine = super(SQLAlchemySessionContextFactory,
-            self).make_object_for_context(name, server_span)
+            self).make_object_for_context(name, span)
         session = Session(bind=engine)
-        server_span.register(SQLAlchemySessionServerSpanObserver(session))
+        span.register(SQLAlchemySessionSpanObserver(session))
```

Comment:
Ah, I see your point. Definitely easy to shoot yourself in the foot if you don't know that local spans create another session. Better to stick to one request -> one session.